src/styles/main.scss

```diff
@@ -1,3 +1,239 @@
+@font-face {
+  font-family: "Roboto";
+  src: url(Roboto/Roboto-Regular.ttf);
```

Comment:
Change the font URLs to be wrapped in quotes, e.g. url('Roboto/Roboto-Regular.ttf')